### PR TITLE
109 use dollars and cents

### PR DESCRIPTION
**Pull request by mikelane**
# Description:

The site was built to use cents for input of money and displayed values in dollars and cents. This update fixes that by implementing django-money. Be aware, there are migrations for this that will adjust the stored monetary value in the database from cents to dollars and cents, too. The migration has been tested in both the forward and reverse direction.

Closes: #109 

## Notes:

I have also upgraded django to version 1.11.18 to address the recent security vulnerability. Additionally, I added django_extensions.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,22 +4,25 @@ verify_ssl = true
...
-Django = "==1.11.3"
+whitenoise = "~=3.3.1"
+django-money = "~=0.14.3"
+django-extensions = "~=2.1.4"
 
 [dev-packages]
 flake8 = '*'
 mypy = '*'
 pytest = '*'
+ipython = "==*"
```
Comment (jarrighi) [inline Pipfile]: Do we need ipython? I'm getting `ERROR: ERROR: No matching distribution found for ipython==*` with it in, but haven't experienced any problems if I take it out.

File: bartercheckout/views.py [hunk 2]
```diff
@@ -4,42 +4,50 @@
 
 import json
 
+from django.http import JsonResponse
 from django.shortcuts import render
```
[the remark this answers is not in the document]
Comment (jarrighi) [reply bartercheckout/views.py]: Wait, no they're not. Ignore me! all good.

File: sistersadmin/settings/dev.py [hunk 3]
```diff
@@ -1,5 +1,3 @@
-from sistersadmin.settings.base import *
-
 # Quick-start development settings - unsuitable for production
```
[the remark this answers is not in the document]
Comment (jarrighi) [reply sistersadmin/settings/dev.py]: Ah, this is fixed in #111

File: Pipfile (hunk 1 above)
Comment (jarrighi) [reply Pipfile]: This also seems to be fixed in #111, so no worries.

File: sistersadmin/settings/dev.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (jarrighi) [reply sistersadmin/settings/dev.py]: so no need to make a change since we can just merge #111 immediately after

**Comment by jarrighi** [comment]
Since the ipython dependency and missing line in settings are fixed in the next PR I'll leave it up to you when you want to add those comments. If you want to add them later, you can merge this and #111 now.

Thanks, @mikelane !

File: Pipfile (hunk 1 above)
Comment (mikelane) [reply Pipfile]: derp. Yeah that was silly.

**Pull request merged**